R/bin.R

```diff
@@ -1,6 +1,6 @@
 bins <- function(breaks, closed = "right",
                  fuzz = 1e-08 * stats::median(diff(breaks))) {
-  if (!is.numeric(breaks)) abort("`breaks` must be a numeric vector")
+  if (!is.numeric(breaks)) cli::cli_abort("{.arg breaks} must be a {.cls numeric} vector")
```

Comment:
Should we update the code style at the same time? (i.e. to put in own `{}` block)

Reply:
I don't have hard feelings - I avoided it to make the PR minimal, but I'm fine going through it all one more time

Reply:
I've now done this